tests/Backend/Snowflake/WorkspacesLoginTypesTest.php

```diff
@@ -99,4 +100,46 @@ public function testWorkspaceCreate(string|null $loginType, bool $async, string
         $this->assertEventWithRetries($this->workspaceSapiClient, $assertCallback, $query);
         $this->assertCredentialsShouldNotWork($connection);
     }
+
+    public function testWorkspaceWithSsoLoginType(): void
+    {
+        $this->initEvents($this->workspaceSapiClient);
+
+        $runId = $this->_client->generateRunId();
+        $this->_client->setRunId($runId);
+        $this->workspaceSapiClient->setRunId($runId);
+
+        $workspaces = new Workspaces($this->workspaceSapiClient);
+        $options = [
+            'loginType' => 'snowflake-person-sso',
+            'backend' => 'snowflake',
+        ];
+        $this->deleteOldTestWorkspaces($this->_client);
+        // we cannot use createWorkspace method as it is calling reset password directly
```

Comment:
no to je ale prusvih pro runner ne? imho bychom meli upravit klienta aby, nedelal reset kdyz prijde v odpovedi loginType SSO. 

jako ono to ted defaultuje na legacy_service, ale i tak

Reply:
Hmm to máš vlastně pravdu :thinking:

Reply:
Upraveno, přidal sem loginType jako enum WorkspaceLoginType, možu to furt změnit protože sem klienta s loginType ještě nevydal :)